lib/resources/r4/health_cards.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name: Health Cards
+field_name_base_url: https://smarthealth.cards/
+fields:
+
+- name: resourceType
+  required: 'Yes'
+  type: string
+  description: The type of the FHIR resource.
+  action: issue
+  example: |
+    {
+      "resourceType": "Parameters"
+    }
+  note: resourceType must be Parameters
+  url: https://hl7.org/fhir/r4/json.html#resources
+- name: parameter
+  required: 'No'
+  type: BackboneElement
+  description: Operation parameter.
+  action: issue
+  children:
+  - name: name
+    required: 'Yes'
+    type: string
+    description: Name from the definition.
+    note: Only credentialType is supported
+    url: https://www.hl7.org/fhir/r4/parameters-definitions.html#Parameters.parameter.name
+    example: |
+      {
+        "parameter": {
+          "name": "credentialType"
+        }
+      }
+  - name: value[x]
+    required: 'No'
+    type: uri
+    description: Value of the parameter.
+    note: |
+      <p>Only the following valueUri value is supported:</p>
+      <ul>
+        <li>https://smarthealth.cards#covid19</li>
```

Comment:
The above states 'only the following is supported' - #covid19 - but in POST body examples, we include #covid19, #health-card, and #immunization. My understanding is that developers should be more specific in their requests as eventually a $health-card-issue for #covid19 will return both a vaccine health card and a lab test health card, right?

Reply:
Added the other two credential types https://github.com/cerner/fhir.cerner.com/pull/659/commits/010373b4f33500144ed55e579fc13e86d741d51a#diff-4ea992907e74c58edf11fe489e39a39f81d9cdd365af93b5bc8e555bbcdf7dd9R42-R43

Reply:
Can you also include a note like (may need @Mturman3  to word-smith):
> *Note*: Cerner recommends using several valueURI at once. This will limit limit the scope of returned values, and will avoid integration issues as Cerner supports further valueURI

Reply:
Added the note here https://github.com/cerner/fhir.cerner.com/pull/659/commits/9a98a7942e70bcc47013fbfbd75a8d35cfc6b1d3#diff-4ea992907e74c58edf11fe489e39a39f81d9cdd365af93b5bc8e555bbcdf7dd9R40